apps/dashboard/src/main/java/com/akto/action/test_editor/SaveTestEditorAction.java

```diff
@@ -289,7 +290,9 @@ public String runTestForGivenTemplate() {
         }
 
         try {
-            GlobalEnums.Severity.valueOf(testConfig.getInfo().getSeverity());
+            if (!TestConfig.DYNAMIC_SEVERITY.equals(testConfig.getInfo().getSeverity())) {
+                GlobalEnums.Severity.valueOf(testConfig.getInfo().getSeverity());
```

Comment:
can't we add another value DYNAMIC_SEVERITY in globalenums under severity?

Reply:
Nope, since the values for severity there are shown in the UI and used actively in documents being stored in the db,  whearas dynamic_severity is only present at the test template level and not used anywhere actively, thus makes more sense to make it a seprate entity.